pkg/router/router_test.go

```diff
@@ -368,15 +369,21 @@ func TestConfigTemplate(t *testing.T) {
 
 	// check the generated config
 	config := filepath.Join(h.workdir, "conf", "haproxy.config")
-	contents, err := ioutil.ReadFile(config)
+	parser := haproxyutil.NewConfigParser(config)
```

Comment:
As an aside I think we should consider using this for our "lobfuscator" ticket - https://issues.redhat.com/browse/NE-419.